eng/pipelines/register-build-jobs.yml

```diff
@@ -0,0 +1,12 @@
+parameters:
+  runtimeRepoAlias: runtime
+  performanceRepoAlias: self
+  buildConfiguration: []
+  mauiFramework: ''
+
+jobs:
+- ${{ each configuration in parameters.buildConfiguration }}:
+  - template: /eng/pipelines/templates/register-build-job.yml@${{ parameters.performanceRepoAlias }}
+    parameters:
+      buildConfiguration: ${{ configuration }}
```

Comment:
buildConfiguration is already a well known term that refers to whether it is compiled in Release or Debug so I think this would be confusing. Maybe `buildType` would be better?